src/transcoding/codec/codecs/libs/vaapi.c

```diff
@@ -151,6 +183,17 @@ static const codec_profile_class_t codec_profile_vaapi_class = {
                 .off      = offsetof(tvh_codec_profile_vaapi_t, device),
                 .list     = tvh_codec_profile_vaapi_device_list,
             },
+            {
+                .type     = PT_BOOL,
+                .id       = "low_power",     // Don't change
+                .name     = N_("Low Power (low_power)"),
```

Comment:
Why is the name repeated in parentheses?

Reply:
the users that use ffmpeg are used with some args (low_power in this case) is the name of the parameter from ffmpeg.
maybe is foolish and we should remove it ... if you think are misleading.

Reply:
Something like this can be mentioned in the docs, it shouldn't be part of the name of that option.

Reply:
good point ... let me remove them.

Reply:
Done